event/src/test/java/com/bazaarvoice/emodb/event/db/astyanax/ChannelAllocationStateTest.java

```diff
@@ -0,0 +1,143 @@
+package com.bazaarvoice.emodb.event.db.astyanax;
+
+import com.google.common.collect.Iterators;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static org.testng.Assert.assertTrue;
+
+public class ChannelAllocationStateTest {
+
+    private static final Logger _log = LoggerFactory.getLogger(ChannelAllocationStateTest.class);
+
+    @Test
+    public void allocateOversizeEvent() {
+        List<Integer> sizes = new ArrayList<Integer>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). (sonar.java.source not set. Assuming 7 or greater.) [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.qa.bazaarvoice.com/coding_rules#rule_key=squid%3AS2293)